Food.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Food.swift
+//  HungryWolfs
+//
+//  Created by Wolfpack Digital on 10.08.2021.
+//
+
+import Foundation
+
+struct Food: Decodable
+{
+    var idCategory: String
+    var strCategory: String
+    var strCategoryThumb: String
+    var strCategoryDescription: String
```

Comment:
Can we change these fields to id, name, thumb and description? Ask Vlad for help as he already done this